common/src/main/java/earth/terrarium/olympus/client/components/dropdown/DropdownScreen.java

```diff
@@ -35,24 +33,15 @@ public int width() {
     }
 
     public int height() {
-        return Math.min(24 * 5, this.dropdown.options().size() * 24) + 3;
+        return Math.min(24 * 5, this.dropdown.options().size() * this.dropdown.getEntryHeight()) + 3;
```

Comment:
you are missing that the height is 5 entries tall, this should always be 5 entries tall even if you change the entry height